beacon_node/lighthouse_network/src/rpc/methods.rs

```diff
@@ -606,6 +606,20 @@ pub enum ResponseTermination {
     LightClientUpdatesByRange,
 }
 
+impl ResponseTermination {
+    pub fn protocol(&self) -> Protocol {
```

Comment:
following the [naming conventions](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv) this could be `as_protocol`.
```suggestion
    pub fn as_protocol(&self) -> Protocol {
```

Reply:
I didn't know the naming conventions. 💡 Thanks!